src/server/game/AuctionHouseBot/AuctionHouseBot.cpp

```diff
@@ -146,7 +146,6 @@ void AuctionBotConfig::GetConfigFromFile()
     SetConfigMax(CONFIG_AHBOT_NEUTRAL_ITEM_AMOUNT_RATIO, "AuctionHouseBot.Neutral.Items.Amount.Ratio", 100, 10000);
 
     SetAHBotIncludes(sConfigMgr->GetStringDefault("AuctionHouseBot.forceIncludeItems", ""));
-    SetAHBotExcludes(sConfigMgr->GetStringDefault("AuctionHouseBot.forceExcludeItems", ""));
```

Comment:
> Which wasn't lost before with tdb updates since those entries are in the conf file.

Right which is why I was suggesting the `disables_custom` table. Users could enter their IDs here just as they used to do in the CONF and not risk losing all their data and without the necessity of maintaining a SQL file somewhere.

It's pretty trivial for the prepared statement to be updated to read from two tables.